basedosdados_api/api/v1/views.py

```diff
@@ -276,6 +278,12 @@ def get(self, request, *args, **kwargs):
                                 (update.frequency, update.entity.slug)
                             )
                             added_updates.add(update.id)
+
+            # Filter by update frequencies
+            update_frequencies: List[Tuple[int, str]] = [
```

Comment:
No retorno de nome, quando for plural tipo `2 month`, deveria retornar no plural `2 meses`. Melhor fazer isso aqui ou retornar via backend direto?